packages/expo-analytics-segment/CHANGELOG.md

```diff
@@ -19,7 +21,7 @@ _This version does not introduce any user-facing changes._
 - Upgraded Segment Analytics iOS to 4.0.4. **This removes the IDFA code that was previously included with the Segment library.** If you would like to collect the IDFA, you must be in the bare workflow or use SDK < 39. ([#9606](https://github.com/expo/expo/pull/9606) by [@cruzach](https://github.com/cruzach/)).
 - The `options` argument for `Segment.alias` now accepts context configuration as well as integration configuration. Previously, this expected just the `integrations` configuration. ([#9606](https://github.com/expo/expo/pull/9606) by [@cruzach](https://github.com/cruzach/)). The expected format now is:
 
-```js
+````js
```

Comment:
```suggestion
```js
```
There is an error in this file in line numer 36.
For some reason there's sth like this `` ```### 🎉 New features`` and thus markdown writer assumes it should use four backticks instead of three. Do you mind fixing this? 🤔 see https://jonschlinkert.github.io/remarkable/demo/#md64=eyJzb3VyY2UiOiJcbiMjIyBCYWNrdGlja3MgaW4gYmFja3RpY2tzIGV4YW1wbGVcblxuPiBVc2UgbW9yZSB0aGFuIDMgdG8gc3RhcnQgdGhlIGV4YW1wbGUsIHRoZW4gb25seSB1c2UgMyBpbnNpZGUgdGhlIGV4YW1wbGUuXG5cbmBgYGBcbnNvbWV0aGluZyBoZXJlXG5cbmBgYFxubW9yZSBzdHVmZlxuYGBgXG5cbmBgYGBcbiIsImRlZmF1bHRzIjp7Imh0bWwiOmZhbHNlLCJ4aHRtbE91dCI6ZmFsc2UsImJyZWFrcyI6ZmFsc2UsImxhbmdQcmVmaXgiOiJsYW5ndWFnZS0iLCJsaW5raWZ5Ijp0cnVlLCJ0eXBvZ3JhcGhlciI6dHJ1ZSwiX2hpZ2hsaWdodCI6dHJ1ZSwiX3N0cmljdCI6ZmFsc2UsIl92aWV3IjoiaHRtbCJ9fQ==

Reply:
Oh, that's interesting 🤔 Good catch!